.yamato/mobile-build-and-run.yml

```diff
@@ -14,7 +14,7 @@ Build_Player_With_Tests_iOS_{{ project.name }}_{{ editor }}:
 
     commands:
         - pip install unity-downloader-cli==1.2.0 --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
-        - unity-downloader-cli -c Editor -c iOS -u {{ editor }} --fast --wait
+        - unity-downloader-cli -c Editor -c iOS -u 2021.3.15f1 --fast --wait
```

Comment:
I can see this laying around and us forgetting to remove this. What's your plan so we don't forget to revert?

Reply:
Creating a high priority ticket in our backlog and linking it to the tools ticket for the bug on their side

Reply:
Here is the ticket: https://jira.unity3d.com/browse/MTT-6201